octa/chain.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "octa",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Octa",
+  "chain_id": "octa",
+  "bech32_prefix": "octa",
```

Comment:
Please register this prefix.
https://github.com/satoshilabs/slips/blob/master/slip-0173.md